starknet/src/types/proposal.cairo

```diff
@@ -43,13 +46,6 @@ struct Proposal {
     active_voting_strategies: u256
 }
 
-// Required for the proposal derivation. Ideally, ContractAddress would impl Default in the corelib.
-impl ContractAddressDefault of Default<ContractAddress> {
```

Comment:
might make more sense to put this in a dedicated file like `defaults.cairo` in `/utils` as its kind of random here. We have `into.cairo` which  serves similar role for the `Into` trait (impls of standard lib types that are not (yet) in the standard lib).

Reply:
true! as I said, ideally, ContractAddress would have a default impl in core lol

7acb81eaa22fab7371b982c8da3358fc70c570d4

Reply:
f4d6f9d691f88a518badafa7eac0175cada8fbcc